backend/coreapp/compilers.py

```diff
@@ -263,13 +263,25 @@ def preset_from_name(name: str) -> Optional[Preset]:
 )
 
 # PS1
-GCC263_MIPSEL = GCCPS1Compiler(
-    id="gcc2.6.3-mipsel",
+PSYQ_MSDOS_CC = (
+    'cpp -P "$INPUT" | unix2dos > ${OUTPUT}c && cp ${COMPILER_DIR}/* . && '
```

Comment:
Unfortunately I do not think I can. The parameter `-K .` mounts the current directory as `G:\` and there is no way for dosemu2 to have any visibility of what is outside that directory. That is also the reason why I hardcoded the name `object.o` rather than using `$OUTPUT`.

Reply:
Ah yes, it's the path to the compiler rather than the input file that is the problem.